### DISCUSSION TITLE
Antivirus malware reports on recent versions

### DISCUSSION THREAD

**Discussion by DropSnorz** [michaelwillis/dragonfly-reverb#152] [General]
Hello,

First of all, thank you for your work on dragonfly plugins suite for years now 😊 

Some common antivirus vendors flag windows binaries as malware since version 3.2.6. These false positives are probably linked to the wine/mingw-gcc based compilation. 

Latest Virus Total reports:
* [ :x: 9 flags] - Version **3.2.8** : https://www.virustotal.com/gui/file/6f1e115f2b626acd35a02b0a6a68fde368d1de29b5e46f58280689ae86457575
* [ :x: 11 flags] - Version **3.2.7** : https://www.virustotal.com/gui/file/e8aeb6dbedee7fc70007b99ccb2ed365bfbf1f83e69f3ea3e6d06261b11568a3
* [ :x: 8 flags] - Version **3.2.6** : https://www.virustotal.com/gui/file/8e404693494ca08b306e2e16e88b8bde0a467099b2ec96564a365e33906c2dc0/
* [ ✔️ 0 flags] - Version **3.2.5** : https://www.virustotal.com/gui/file/440c634b6dc658a5b1784e88115297b34dfbfdf1a8461af9e53ef8d3c1f2186e

Don't know if there is something we can do to get rid of these false positives.

**Comment by falkTX** [comment]
seeing as these are false positives, I would just stop trusting such antiviruses right away

**Comment by michaelwillis** [comment]
I'm not certain of any way to mitigate false positive from these malware products.

**Comment by DropSnorz** [comment]
FYI, I've restarted the analysis on the same version, and I can't see those false positives anymore from common AV vendors.
